pants

```diff
@@ -137,10 +141,27 @@ function bootstrap_pants {
   echo "${PANTS_BOOTSTRAP}/${target_folder_name}"
 }
 
+function create_pants_ini_if_missing {
+  if [[ -f "pants.ini" ]]; then
+    return 0
+  fi
+  log "Config file \`pants.ini\` not detected in the repository root. Creating with sensible defaults.\n"
+  touch "pants.ini"
+  pants_version="$(./pants -V)"
+  log "* Pinning \`pants_version\` to the most recent stable release: ${pants_version}."
+  cat <<EOF >> "pants.ini"
+[GLOBAL]
+pants_version: ${pants_version}
+EOF
+  green "\n\`pants.ini\` created. You may now run \`./pants\`."
+  exit 0
```

Comment:
Calling `exit` within a function unless it's something like `die` can be a bit hard to follow. I think below (instead of `create_pants_ini_if_missing`) could be something like:

```bash
if [[ ! -f 'pants.ini' ]]; then
  autogen_pants_ini
  exit 0
fi
```
and then we can remove the `exit` here and the check at the top of this method too.